lib/LaTeXML/Core/Gullet.pm

```diff
@@ -367,7 +367,6 @@ sub readBalanced {
   my ($self, $expanded) = @_;
   my @tokens = ();
   my ($token, $level) = (undef, 1);
-  my $startloc = $self->getLocator;
```

Comment:
I am not too happy about losing the accuracy here... maybe we could use the locator of the first read token instead...

Reply:
Knowing where a failed balanced sequence started is very helpful for debugging. My first thought was maybe could depend on verbosity, but likely the cost of repeatedly looking up the switch may cancel the benefit.

Reply:
Actually the verbosity flag is completely workable, it is much much cheaper to execute a conditional, compared to calling `getLocator`.